### PR TITLE
add project source code

### PR DESCRIPTION
**Pull request by Finniki**
# Description

This PR adds the HTML and CSS source codes for this project.

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Details

- ### **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Feature

- ### **What is the current behavior?** (You can also link to an open issue here)
Displays content and styling for the user to view

## Screenshots (if applicable)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by gertie-sheshe** [comment]
Great job! 👏 @Finniki Congratulations for putting the effort to complete this! 

What I liked:
1. Usage of CSS grid to solve the challenge is 👌 
2. Proper observation of accessibility (i.e semantic HTML) 🔥
3. It's responsive! 👏 

A few things to address :)
1. I think you forgot to push the images folder. Currently the logos do not show:
Screenshot 2022-09-18 at 19 12 49
2. Responsiveness work great! 💪🏾 But the background color is not the correct one. The text description is meant to occupy more screen real estate and the green border shouldn't be there:
Screenshot 2022-09-18 at 19 17 34
3. Medium screens (i.e iPads) responsiveness is not completely there yet :) You can have it look like the mobile screen with everything in column layout:
Screenshot 2022-09-18 at 19 04 21

**Comment by Finniki** [comment]
Oh! @gertie-sheshe You are right. Indeed, I have omitted the images folder. How silly of me!
Yes, I see the other concerns and I will make the corrections (but after watching Big Brother 🌚). To be honest, I get tired of styling pretty quickly and tend to rush through my media queries. I use background colors and borders to get the layout right and must have forgotten to remove them afterwards. I will take care to be more thorough in the future. 
Thanks for taking the time to review my codes and give feedback. 

Some questions;
1. I think a 2 by 2 grid would look nice for the medium screens, what do you think?
2. In a case like this (and for future reference), do I go ahead and merge this PR then fix the issues in a second PR or do I close this one down and recreate the PR after correcting the files?

**Comment by gertie-sheshe** [comment]
> Oh! @gertie-sheshe You are right. Indeed, I have omitted the images folder. How silly of me! Yes, I see the other concerns and I will make the corrections (but after watching Big Brother 🌚). To be honest, I get tired of styling pretty quickly and tend to rush through my media queries. I use background colors and borders to get the layout right and must have forgotten to remove them afterwards. I will take care to be more thorough in the future. Thanks for taking the time to review my codes and give feedback.
> 
> Some questions;
> 
> 1. I think a 2 by 2 grid would look nice for the medium screens, what do you think?
> 2. In a case like this (and for future reference), do I go ahead and merge this PR then fix the issues in a second PR or do I close this one down and recreate the PR after correcting the files?

I definitely understand about styling and getting bored 😂 I struggle with that too! But it is a required skill so we push through to get it right, and the result afterwards is good

Answers to the questions:
1. Ah yeah, I think a 2 x 2 will definitely look good!
2. No, you continue working on this PR. Just make the changes and continue pushing to this branch and I will keep reviewing. When all is good, then you can proceed to merge

Lol, enjoy your BB episodes @Finniki  - I'll be looking forward to the fixes later today

**Comment by Finniki** [comment]
Hi @gertie-sheshe I have implemented changes to address the issues you pointed out.. Please help review it again. I see +244 -0 at the top there, not sure what the -0 means. I hope it has nothing to do with the end-of-file line you talked about on my previous project.. I made sure to add the extra line at the end of each file, so if you could clarify what that means.
Thanks a lot!

**Comment by Finniki** [comment]
Hi @gertie-sheshe just wanted to remind you to review this PR also. Thank you

File: index.html [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    <div class="card">
+      <header>
+        <h1>Reliable, efficient delivery</h1>
+        <h2>Powered by Technology</h2>
+        <p>
+          Our Artificial Intelligence powered tools use millions of project data
+          points to ensure that your project is successful
+        </p>
+      </header>
+      <div class="content">
```
Comment (gertie-sheshe) [inline index.html]: ``
```suggestion
      <main class="content">
```

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+  opacity: 0.8;
+  font-size: 12px;
+  letter-spacing: 0.5px;
+}
+img {
+  padding: 30px 0px 0px 190px;
+  width: 50px;
+}
+
+@media screen and (min-width: 370px) and (max-width: 576px) {
```
Comment (gertie-sheshe) [inline styles.css]: Because of this line, all screens below 370px width will render the desktop styles :)

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+  grid-area: 3/2/5/3;
+  border-top: 3px solid yellow;
+}
+.calculator {
+  grid-area: 2/3/4/4;
+  border-top: 3px solid rgb(0, 81, 128);
+}
+.section {
+  box-shadow: 5px 10px 10px lightgray, -5px 0px 10px lightgray;
+  padding: 0px 20px;
```
Comment (gertie-sheshe) [inline styles.css:61]:
```suggestion
  padding: 0px 20px;
  height: fit-content;
```

**Review by gertie-sheshe — COMMENTED** [review]
Looks 🔥🔥 @Finniki Thanks for addressing the previous changes 🙌🏾

I have added a few comments and some suggested code changes. Here are some screenshots of what the code suggestions and comments refer to:

height
media-query


For the `+244 -0` That is Github tracking your file changes. So in this case you have added 244 lines and have not deleted any code. Hope that makes sense? You can read more [here](https://stackoverflow.com/questions/38346919/what-does-the-two-number-mean-in-github-contributor-graph-page)

File: styles.css (hunk 2 above)
Comment (Finniki) [reply styles.css]: Just gonn take out the min-width then..

**Comment by Finniki** [comment]
Done @gertie-sheshe 
All issues corrected and ready for final review ijn

**Pull request merged**